Lib/inspect.py

```diff
@@ -517,7 +526,12 @@ def iscode(object):
         co_names            tuple of names other than arguments and function locals
         co_nlocals          number of local variables
         co_stacksize        virtual machine stack space required
-        co_varnames         tuple of names of arguments and local variables"""
+        co_varnames         tuple of names of arguments and local variables
+        co_qualname         fully qualified function name
+
+        co_lines            returns an iterator that yields successive bytecode ranges
+        co_positions        returns an iterator of source code positions for each bytecode instruction
+        replace             return a copy of the code object with a new values"""
```

Comment:
```suggestion
        replace             returns a copy of the code object with a new values"""
```

Or as Victor suggested, use `co_lines()`, `co_positions()` and `replace()`